boxsdk/auth/jwt_auth.py

```diff
@@ -360,3 +361,51 @@ def _normalize_rsa_private_key_passphrase(passphrase):
                 .format(passphrase.__class__.__name__)
             )
         return passphrase
+
+    @classmethod
+    def from_config_dictionary(cls, config_dictionary):
+        """
+        Create an auth instance as defined by the given configuration dictionary.
+
+        The dictionary should have the following structure:
```

Comment:
Rather than documenting the structure here, maybe just point them to the place to find this in the developer app console.

I don't think we want users to think that we're encouraging them to manually construct these JSON objects. At that point, they're better off calling the constructor manually.

Also, link to any documentation that there might be about this feature.